src/main/resources/flyway/sql/V3__add_description.sql

```diff
@@ -1,2 +0,0 @@
-ALTER TABLE riff
```

Comment:
Did this flyway already run on a DB ? if so, should you re-alter the table to remove it back ?

Reply:
Yes, it has already run on dev and qa instance. We need to revert to a previous snapshot on these instances